src/display.cpp

```diff
@@ -1033,7 +1033,9 @@ void display::get_terrain_images(const map_location& loc, const std::string& tim
 {
 	terrain_image_vector_.clear();
 
-	std::vector<image::light_adjust> lighting;
+	static std::vector<image::light_adjust> lighting;
+	lighting.clear();
```

Comment:
What‘s the point of this?

Reply:
We already keep a permanent vector in the class for terrain images (`terrain_image_vector_`). I figured it was a safe bet to keep a static vector for lightmap data, since this function is called so often. But I can do more extensive testing.